pennylane/_qubit_device.py

```diff
@@ -1316,7 +1317,12 @@ def expval(self, observable, shot_range=None, bin_size=None):
         # exact expectation value
         if self.shots is None:
             try:
-                eigvals = self._asarray(observable.eigvals(), dtype=self.R_DTYPE)
+                eigvals = self._asarray(
+                    observable.eigvals()
+                    if not isinstance(observable, MeasurementValue)
+                    else np.arange(2 ** len(observable.wires)),
```

Comment:
Is `len(observable.wires)` allowed to be anything other than 1 now? And could the `MeasurementValue` just have eigenvalues [0, 1]?

Reply:
`MeausementValue` is not an operator or measurement process, so I don't think it makes sense to add an `eigvals` property to it. It's easier to just use `arange(2**num_wires)`.